### PR TITLE
Cleanup next-task page

### PR DESCRIPTION
**Pull request by cas--**
Also fixed link the old `Product Backlog` label.

### CODE REVIEW

File: next-task.md [hunk 1]
```diff
@@ -1,27 +1,26 @@
...
 
 ## DevOps and Support
 
-First focus is that our systems are running smoothly and that all our customers' issues have been resolved and answered. If that is done, next thing would be to make sure our documents are updated. Read through the [Handbook](https://github.com/niteoweb/handbook) and [Intra](https://intra.niteoweb.com/) and review if there are any updated necessary. 
-
-If you are DevOps and have had no tasks for some time, you can join the next sprint.
-
-If you are Support and have had a slow few days, start reading on how we could improve our support. This may include improving our processes or rewriting our responses.
+ - First focus on whether our systems are running smoothly and that all our customers' issues have been resolved and answered. 
+ - Make sure our documents are updated. Read through the [Handbook](https://github.com/niteoweb/handbook) and [Intra](https://intra.niteoweb.com/) and review if there are any updated necessary.
```
Comment (zupo) [inline next-task.md]: updated -> updates

Comment (cas--) [reply next-task.md]: Rewritten this line

**Comment by zupo** [comment]
Please squash commits.

**Comment by cas--** [comment]
I can do that on the rebase and merge iirc, as this is github created branch

**Comment by zupo** [comment]
TIL: ^ :)

**Comment by cas--** [comment]
Ah it's only 'sqaush and merge' :cry: 

I'll sort this though

**Pull request merged**